src/test/java/tests/waiter/WaiterOrderStatusTest.java

```diff
@@ -0,0 +1,142 @@
+package tests.waiter;
+
+import org.testng.Assert;
+import org.testng.annotations.*;
+import pages.HomePage;
+import pages.SignInPage;
+import pages.waiter.WaiterAssignedWaiterPage;
+import pages.waiter.WaiterHistoryPage;
+import pages.waiter.WaiterInProgressPage;
+import pages.waiter.WaiterMainPage;
+import tests.BaseTest;
+
+public class WaiterOrderStatusTest extends BaseTest    {
+
+    /*
+    ***************************************************
+
+            Fields used without preconditions
+
+    ***************************************************
+    */
+
+    private final String waiterEmail = "alexandriawright@test.com";
+    private final String waiterPassword = "1";
+
+    private final String firstOrderNumber = "№138";
+    private final String secondOrderNumber = "№123";
+
+    /*
+    ************************************************
+        End of fields used without preconditions
+    ************************************************
+    */
+
+    private SignInPage signInPage;
+    private WaiterMainPage waiterMainPage;
+
+    @BeforeClass
+    public void preconditions() {
+        HomePage homePage = new HomePage(driver);
+        signInPage = homePage
+                .getHeaderGeneralPageComponent()
+                .signInAccess();
+    }
+
+    @BeforeMethod
+    public void waiterLogIn()    {
+        signInPage.setUserEmailInputField(waiterEmail);
+        signInPage.setUserPasswordInputField(waiterPassword);
+        signInPage.clickSignInBtn();
+        waiterMainPage = new WaiterMainPage(driver);
+    }
+
+    @Test
+    public void waiterOrderStatusTest601()  {
+        logger = extent.createTest("Waiter order status test 6.01");
+        String orderStatus;
+
+        WaiterAssignedWaiterPage waiterAssignedWaiter = waiterMainPage
+                .waiterHeader
+                .waiterAssignedWaiterPageAccess();
+
+        waiterAssignedWaiter.orders.expandOrder(firstOrderNumber).startOrderBtnClick();
+
+        WaiterInProgressPage waiterInProgressPage = waiterAssignedWaiter
+                .waiterHeader
+                .waiterInProgressPageAccess();
+
+        orderStatus = waiterInProgressPage.orders.getOrderStatus(firstOrderNumber);
+
+        Assert.assertEquals(orderStatus, "In progress");
```

Comment:
is it correct the order of parameters?

Reply:
yes - first parameter is actual result and the second one is expected result